### PR TITLE
[1LP][RFR] Transfer tests to jhenner.

### PR DESCRIPTION
**Pull request by jaryn**
There is naming clash of centralized control (at the moment also
often called Replication) with HA tests, marked as Replication as well.
I belive these are very different features. So I've:

 * Created new polarion marker for the Ha proxy tests.
 * Set polarion markers have been set on the tests.
 * Set the casecomponent to HAProxy on HA proxy related tests.

### CODE REVIEW

File: cfme/test_requirements.py [hunk 1]
```diff
@@ -189,6 +189,12 @@ def test_quota_alert():
     assignee_id='pvala',
 )
 
+ha_proxy = pytest.mark.requirement(
+    "Database HA proxy",
+    description='Databse HA proxy and replication',
+    assignee_id='jhenner',
+)
```
Comment (john-dupuy) [inline cfme/test_requirements.py]: @jaryn Should we also remove the `replication` test requirement? Or are there still tests using the `replication` requirement?

File: cfme/test_requirements.py [hunk 2]
```diff
@@ -189,6 +189,12 @@ def test_quota_alert():
     assignee_id='pvala',
 )
 
+ha_proxy = pytest.mark.requirement(
+    "Database HA proxy",
+    description='Databse HA proxy and replication',
```
Comment (mshriver) [inline cfme/test_requirements.py]: typo, missing an a in `Databse`

File: conf/polarion_tools.yaml [hunk 3]
```diff
@@ -131,6 +131,7 @@ docstrings:
             - Provisioning
             - Quota
             - Replication
+            - HAProxy
```
Comment (mshriver) [inline conf/polarion_tools.yaml]: I have added this casecomponent to Polarion, please don't change this string.

File: cfme/test_requirements.py (hunk 1 above)
Comment (jaryn) [reply cfme/test_requirements.py]: All tests that are left with replication marker should be mdnadeems. I am sending him a request for renaming/updating the description to say these are centralized-admin tests.

Comment (mnadeem92) [reply cfme/test_requirements.py]: @jaryn  Replication FA is owned by Devidas Gaikwad

@dgaikwad , Could you please have a look into it.

Comment (dgaikwad) [reply cfme/test_requirements.py]: @jaryn Databse HA proxy and  replication are two different focus area. I think you can remove `replication `string from `ha_proxy `description.

Comment (jaryn) [reply cfme/test_requirements.py]: > @jaryn Databse HA proxy and replication are two different focus area. I think you can remove `replication `string from `ha_proxy `description.

@dgaikwad For HA you need to have data replicated (https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/high_availability_guide/index#configuring_primary_db). But by my understanding it is quite different kind of replication than that is done in your focus area which I believe should be named Centralized administration (https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/deployment_planning_guide/index#central-administration)

Some replication is done in both focus areas.

File: cfme/test_requirements.py (hunk 2 above)
Comment (jaryn) [reply cfme/test_requirements.py]: fixed

File: conf/polarion_tools.yaml (hunk 3 above)
Comment (jaryn) [reply conf/polarion_tools.yaml]: OK

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**